src/storage/txn/commands/resolve_lock_readphase.rs

```diff
@@ -27,8 +27,10 @@ command! {
 impl CommandExt for ResolveLockReadPhase {
     ctx!();
     tag!(resolve_lock);
-    command_method!(readonly, bool, true);
-    command_method!(write_bytes, usize, 0);
+    property!(readonly);
+    fn write_bytes(&self) -> usize {
+        0
+    }
```

Comment:
Give it padding lines above and below it?

Reply:
Done.